engines/griffon/dialogs.cpp

```diff
@@ -404,12 +453,20 @@ void GriffonEngine::configMenu() {
 					cursel--;
 					if (cursel < MINCURSEL)
 						cursel = MAXCURSEL;
+					if (ttsMan != nullptr && ConfMan.getBool("tts_enabled")) {
+						ttsMan -> say(optionTitles[curselMapTitles[cursel]]);
+						ttsMan -> say(optionValues[curselMapValues[cursel]], Common::TextToSpeechManager::QUEUE);
+					}
```

Comment:
This block of code, and the similar one below, does not quite work properly with the `Exit + Save` and `Exit` menus for me when I use the arrow keys quickly to go down or up in the menu. I think the issue is with calling `say()` with an empty text when `curselMapTitles[cursel]` is 0, and with queuing the following text.
What seems to work properly is to check that there is some text to speak, and only use queuing when there was an interrupt before. That would give something like this:
```c++
if (ttsMan != nullptr && ConfMan.getBool("tts_enabled")) {
	Common::TextToSpeechManager::Action valueAction = Common::TextToSpeechManager::INTERRUPT;
	if (curselMapTitles[cursel] != 0) {
		ttsMan->say(optionTitles[curselMapTitles[cursel]]);
		valueAction = Common::TextToSpeechManager:: QUEUE;
	}
	if (curselMapValues[cursel] != 0)
		ttsMan->say(optionValues[curselMapValues[cursel]], valueAction);
}
```

Also note that we are not expecting a space before and after `->` (so it should be `ttsMan->say` and not `ttsMan -> say`).